lambdas/address/src/main/java/uk/gov/di/ipv/cri/address/api/handler/AddressHandler.java

```diff
@@ -75,6 +75,7 @@ public APIGatewayProxyResponseEvent handleRequest(
             // If we have at least one address, we can return a 201 with the authorization code
             if (!addresses.isEmpty()) {
                 SessionItem session = sessionService.validateSessionId(sessionId);
+                eventProbe.log(Level.INFO, "found session");
```

Comment:
Doesn't the `AddressHandlerTest` need mock behaviour for the event probe like the other handlers?

Reply:
@techaman-m I  think this test does set up mockito behaviour - maybe it was missing when you reviewed